README.md

```diff
@@ -20,15 +20,23 @@ Use the command:
 
 `podman machine start`
 
-4. Build the container image
+4. Change the Containerfile's "FROM" directive to match your architecture. 
+
+Uncomment the relevant FROM statement at the top of the Containerfile
+
+5. Edit the .env.templ file to uncomment the ALLOWED_HOSTS and STATIC_ROOT options 
```

Comment:
The `.env.templ` file is a tracked file and is meant to be a template for folks using the project. I would recommend updating the instructions to have the user _copy_ the `.env.templ` file to `.env` and then adjust as necessary. This will allow the developer to then update the configuration as necessary without changing the template file.